<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Catamaran dashboard</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Navigation</string>
       </attribute>
       <widget class="QCheckBox" name="joystickCheckbox">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>20</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Use Joystick</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="rightSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>270</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
       <widget class="QSlider" name="leftSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>90</y>
          <width>19</width>
          <height>161</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
       <widget class="QSpinBox" name="leftSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>270</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QSlider" name="rightSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>90</y>
          <width>19</width>
          <height>160</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>160</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>160</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>80</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>100</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>80</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>100</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>240</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>-100</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>240</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>-100</string>
        </property>
       </widget>
       <widget class="QSlider" name="speedSlider">
        <property name="geometry">
         <rect>
          <x>870</x>
          <y>80</y>
          <width>20</width>
          <height>141</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
       <widget class="QSlider" name="directionSlider">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>250</y>
          <width>160</width>
          <height>19</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
       <widget class="QSpinBox" name="speedSpinBox">
        <property name="geometry">
         <rect>
          <x>910</x>
          <y>140</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="directionSpinBox">
        <property name="geometry">
         <rect>
          <x>970</x>
          <y>250</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>70</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>100</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>140</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>280</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>210</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>-100</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>280</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>-100</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>950</x>
          <y>280</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>100</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>230</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Direction</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>100</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Speed</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>60</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Left engine</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>60</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Right engine</string>
        </property>
       </widget>
       <widget class="QPushButton" name="stopBtn">
        <property name="geometry">
         <rect>
          <x>930</x>
          <y>320</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="lightBtn">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>20</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Light</string>
        </property>
       </widget>
       <widget class="QPushButton" name="honkBtn">
        <property name="geometry">
         <rect>
          <x>950</x>
          <y>20</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Honk</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>551</width>
          <height>431</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="zoomInBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/zoom.png</normaloff>:/images/zoom.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
       <widget class="QToolButton" name="zoomOutBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/zoom.png</normaloff>:/images/zoom.png</iconset>
        </property>
       </widget>
       <widget class="QSlider" name="zoomSlider">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>19</width>
          <height>160</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
       <widget class="QGroupBox" name="waypointGroupBox">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>360</y>
          <width>411</width>
          <height>80</height>
         </rect>
        </property>
        <property name="title">
         <string>Waypoints</string>
        </property>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>311</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Right-click on map to set a way point (WP)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="clearWpBtn">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>50</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Clear all WPs</string>
         </property>
        </widget>
        <widget class="QPushButton" name="removeWpBtn">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Remove last WP</string>
         </property>
        </widget>
        <widget class="QPushButton" name="startNavigationWpBtn">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>50</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </widget>
       <widget class="QComboBox" name="navModeComboBox">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>20</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Plots</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Configuration</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>700</width>
            <height>100</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>PT100 module1</string>
             </property>
             <widget class="QComboBox" name="pt100_module1_comboBox">
              <property name="geometry">
               <rect>
                <x>130</x>
                <y>20</y>
                <width>69</width>
                <height>22</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_18">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>20</y>
                <width>81</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>Sampling mode</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_19">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>50</y>
                <width>101</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>Sampling period (ms)</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="lineEdit">
              <property name="geometry">
               <rect>
                <x>130</x>
                <y>50</y>
                <width>71</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>200</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>PT100 module2</string>
             </property>
             <widget class="QComboBox" name="pt100_module2_comboBox">
              <property name="geometry">
               <rect>
                <x>130</x>
                <y>20</y>
                <width>69</width>
                <height>22</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_22">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>20</y>
                <width>81</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>Sampling mode</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_23">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>50</y>
                <width>101</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>Sampling period (ms)</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="lineEdit_3">
              <property name="geometry">
               <rect>
                <x>130</x>
                <y>50</y>
                <width>71</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>200</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Plots</string>
             </property>
             <widget class="QLabel" name="label_17">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>20</y>
                <width>61</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>Nb of graphs</string>
              </property>
             </widget>
             <widget class="QSpinBox" name="graphNbSpinBox">
              <property name="geometry">
               <rect>
                <x>100</x>
                <y>20</y>
                <width>42</width>
                <height>22</height>
               </rect>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="value">
               <number>3</number>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
          <zorder>groupBox_3</zorder>
          <zorder>groupBox_2</zorder>
          <zorder>groupBox</zorder>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="addressesContainer" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Sensors</string>
       </attribute>
       <widget class="QTreeView" name="availableSensorsInput">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>256</width>
          <height>411</height>
         </rect>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>true</bool>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QTableView" name="registeredSensors">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>31</y>
          <width>681</width>
          <height>411</height>
         </rect>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
       <widget class="QPushButton" name="addSensorFromList">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>220</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>=&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Output</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="statusText">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>tabWidget</zorder>
   <zorder>statusText</zorder>
   <zorder>statusLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
